C/CImPlot/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CImPlot"
+version = v"1.76.0"
+
+# Collection of sources required to build CImGui
+sources = [
+    GitSource("https://github.com/ocornut/imgui.git",
+              "5503c0a12e0c929e84b3f61b2cb4bb9177ea3da1"),
+
+    GitSource("https://github.com/cimgui/cimplot.git",
+              "8799c69b20081a744f44bb1ff9d39f6c8d8f23a1"),
+
+    GitSource("https://github.com/epezent/implot.git",
+              "1353014bce7d330e612529cb6193d811281eabac"),
+
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mv cmake/CMakeLists.txt ./
+mv implot cimplot/
+mkdir build && cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+make -j${nproc}
+make install
+install_license ../cimplot/LICENSE ../cimplot/implot/LICENSE
```

Comment:
Uhm, these two files have the same name, so they are overwriting each other.  I fear you need to manually copy them to `${prefix}/share/licenses/${SRC_NAME}/` with two different names.  You also need to create the destination directory

Reply:
Individual licenses should be preserved now. For good measure I included imgui's as well.